### ISSUE TITLE
name

### ISSUE THREAD

**Issue by nieknooijens** [guusw/unnamed-sdvx-clone#8]
Hi there

it's still "unnamed sdvx (soundvoltex) clone"
shouldn't we make it something like "openSDVX" (just like openITG)?

**Comment by Drewol** [comment]
Should maybe not use 'SDVX' at all because Konami

**Comment by nieknooijens** [comment]
yeah, konami is a bitch... just read they forced the creator of kshootmania to implement DRM into his program. I contacted them that I wanted to import soundvoltex but even if offered over €6000,- they won't bodge. They constantly just tell you it's "Japan only". that's why I really want this to succeed, having an opensource soundvoltex-game letting them know that they really should listen to their fans in stead of sueing them all the time.

**Comment by nieknooijens** [comment]
since the scrolling somewhat resembles guitar hero, what about effects hero? :wink: 

btw, I made a little lets-play video:
https://www.dropbox.com/s/luk6lbw1362fwh1/letsplay1.mp4?dl=0

**Comment by electromuis** [comment]
PPPW

(Pew Pew Pew Woosh)
After the sound effects of 3 notes following a laser
